src/actions/otherActions.tsx

```diff
@@ -45,6 +45,18 @@ function restoreMapView(mapState: PersistedMapState) {
   }
 }
 
+export function getHiddenElements(element: HTMLElement | null) {
+  if (!element) return [];
+
+  return [
+    // Map Elements
+    element.querySelector(".ol-unselectable.ol-control.MuiBox-root.css-0"),
+    element.querySelector(".ol-zoom.ol-unselectable.ol-control"),
+    // Statistics-Panel Elements
+    element.querySelector("#statistics-row-buttons"),
+  ].filter(Boolean) as HTMLElement[];
+}
+
```

Comment:
I'd prefer we had that function in two versions defined where they are needed and selecting the elements that are applicable only for the container which shall be exported.